### PR TITLE
Transpile the library so it can be used by browsers

### PR DESCRIPTION
**Pull request by trevorblades**
This pull request brings in Babel to help transpile the library into browser-ready code. To recap my changes, I've:
 - Installed `babel-cli` and `babel-preset-env`. This also creates a `package-lock.json` file because I'm using `npm>5` (More info on package locks here: https://docs.npmjs.com/files/package-locks). I also set up a simple Babel config where we only need to use the `env` preset.
 - Added a `prepare` npm script that runs *automatically* before every `publish`. It transpiles `index.js` to `lib/index.js`.
 - Added the `lib` directory to our files ignored by git. We don't need to maintain the transpiled version of the library in git.
 - Changed the `main` entry point of the package to point to the new, transpiled version at `lib/index.js` and swapped the `index.js` entry in the `files` array to `lib`

Closes https://github.com/Kikobeats/aspect-ratio/issues/9

### CODE REVIEW

**Comment by Kikobeats** [comment]
Althought I appreciated this PR, it sounds a little confuse for me.

Fat arrow is a popular feature so you will find it in a lot of modules, so how to want ot handle that? transpiling every individual dependency?

Could be possible configure your out-of-box build process for transpile the dependencies?

**Comment by trevorblades** [comment]
While fat arrows are popular among many node libraries, they need to be transpiled if we want them to be consumed by browser-based applications. This change doesn't affect the way that you write your code, but it just makes the published library compatible with more platforms and uses.

It's pretty inconvenient for consumers of a library to have to worry about transpiling their node modules on every webpack (or browserify, etc.) build.

Many other browser-compatible libraries use this same method of building before publishing! Take `polished` for example: https://github.com/styled-components/polished/blob/master/package.json#L29

They write all of their modules using new JS syntax and tools, but then build out more compatible versions at publish-time.

File: package.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
     "meow": "latest"
   },
   "devDependencies": {
+    "babel-cli": "^6.26.0",
```
Comment (Kikobeats) [inline package.json]: latest pls

File: package.json [hunk 2]
```diff
@@ -29,6 +29,8 @@
...
+    "babel-preset-env": "^1.6.1",
```
Comment (Kikobeats) [inline package.json]: as well

**Comment by trevorblades** [comment]
@Kikobeats dependencies have been updated to latest 👍

**Comment by trevorblades** [comment]
@Kikobeats just a heads up that you need to be running `npm` version 4 or greater for the `prepare` script to work!

**Comment by Kikobeats** [comment]
done!

**Pull request merged**